src/Gui/Document.cpp

```diff
@@ -1440,9 +1440,9 @@ void Document::RestoreDocFile(Base::Reader &reader)
             try {
                 const char** pReturnIgnore=nullptr;
                 std::list<MDIView*> mdi = getMDIViews();
-                for (std::list<MDIView*>::iterator it = mdi.begin(); it != mdi.end(); ++it) {
-                    if ((*it)->onHasMsg("SetCamera"))
-                        (*it)->onMsg(cameraSettings.c_str(), pReturnIgnore);
+                for (auto & it : mdi) {

```

Comment:
At a quick glance this looks like the same construct as line 1561 -- why not const?

Reply:
ok